actionview/lib/action_view/test_case.rb

```diff
@@ -223,7 +223,7 @@ def setup_with_controller
         @request = @controller.request
         @view_flow = ActionView::OutputFlow.new
         @output_buffer = ActionView::OutputBuffer.new
-        @rendered = +""
+        @rendered = self.class.content_class.new(+"")
```

Comment:
This breaks a compatibility workaround that was documented in the upgrading guide: 

https://github.com/rails/rails/blob/dd6f3e13285cff5e248340f30bcdc713eebed119/guides/source/upgrading_ruby_on_rails.md#L298-L308

Maybe we should go with clearing the ivar, since users may have added that workaround to their apps already.

Reply:
Hmm, that workaround was suggested because the change to `rendered` returned an instance of a class that was not a `String`. 

The inheritance chain has changed to where the `content_class` is a `String` subclass, which might make that guidance no longer necessary.